tests/unit/test_transfer_client.py

```diff
@@ -1412,3 +1411,172 @@ def test_delete_endpoint_acl_rule(self):
+        ddata.add_item(dest_path)
+        delete_id = self.tc2.submit_delete(ddata)["task_id"]
+
+        # sdktester1a gets endpoint manager task list
+        tasks_doc = self.tc.endpoint_manager_task_list(
+            filter_endpoint=GO_EP2_ID,
+            filter_user_id=get_user_data()["sdktester2b"]["id"])
+
+        # confirm submitted tasks can be found
+        # and tasks have some expected fields
+        expected_fields = ["username", "deadline", "type",
+                           "source_endpoint_id"]
+        delete_found = False
+        transfer_found = False
+        self.assertIsInstance(tasks_doc, PaginatedResource)
+        for task in tasks_doc:
+
+            for field in expected_fields:
+                self.assertIn(field, task)
+
+            if task["task_id"] == transfer_id:
+                transfer_found = True
+            if task["task_id"] == delete_id:
+                delete_found = True
+            if transfer_found and delete_found:
+                break
+
+        # fail if both not found
+        self.assertTrue(delete_found and transfer_found)
+
+    def test_endpoint_manager_task_successful_transfers(self):
+        """
+        Has sdktester2b submit a recursive transfer of share/godata to the
+        managed_ep. Waits for the task to complete, then has sdktester1a get
+        the successful transfers of the task as an admin. Confirms all 3 files
+        are seen, and some expected fields are present.
+        """
+        # new dir with randomized name to prevent collision
+        dest_dir = "transfer_dest_dir-" + str(getrandbits(128))
+        dest_path = "/" + dest_dir + "/"
+        self.tc2.operation_mkdir(self.managed_ep_id, dest_path)
+        # transfer a the files
+        tdata = globus_sdk.TransferData(self.tc2, GO_EP1_ID,
+                                        self.managed_ep_id,
+                                        notify_on_succeeded=False)
+        source_path = "/share/godata/"
+        tdata.add_item(source_path, dest_path, recursive=True)
+        task_id = self.tc2.submit_transfer(tdata)["task_id"]
+
+        # track asset for cleanup
+        self.asset_cleanup.append({"function": self.deleteHelper,
+                                   "args": [self.managed_ep_id, dest_path]})
+        # wait for task to complete
+        self.assertTrue(
+            self.tc2.task_wait(task_id, timeout=30, polling_interval=1))
+
+        # sdktester1a gets successful transfers as admin
+        success_doc = self.tc.endpoint_manager_task_successful_transfers(
+            task_id)
+
+        # confirm results
+        self.assertIsInstance(success_doc, PaginatedResource)
+        count = 0
+        for transfer in success_doc:
+
+            self.assertEqual(transfer["DATA_TYPE"], "successful_transfer")
+            self.assertIsNotNone(re.match(dest_path + "file[1-3].txt",
+                                 transfer["destination_path"]))
```

Comment:
This will be my one and only nitpick with this PR: `flake8` doesn't seem to fail on this, but it should.

The indentation doesn't match because `transfer["destination_path"]` is an argument to `re.match`.